.tekton/build-service-pull-request.yaml

```diff
@@ -167,6 +167,71 @@ spec:
       workspaces:
       - name: basic-auth
         workspace: git-auth
+    - name: sealights-go-instrumentation
+      runAfter:
+        - clone-repository
+      taskRef:
+        params:
+        - name: name
+          value: sealights-go-oci-ta
+        - name: bundle
+          value: quay.io/konflux-ci/tekton-catalog/task-sealights-go-oci-ta:0.1@sha256:3ee87d28065205ee5f1e32ec012cac4b167af6525f7504a9657612e4387d0eae
+        - name: kind
+          value: task
+        resolver: bundles
+      params:
+        - name: SOURCE_ARTIFACT
+          value: $(tasks.clone-repository.results.SOURCE_ARTIFACT)
+        - name: go-version
+          value: "1.21.9"
+        - name: sealights-secret
+          value: "sealights-credentials"
+        - name: component
+          value: '{{ repo_name }}'
+        - name: branch
+          value: '{{ source_branch }}'
+        - name: revision
+          value: '{{ revision }}'
+        - name: repository-url
+          value: '{{ repo_url }}'
+        - name: pull-request-number
+          value: '{{ pull_request_number }}'
+        - name: target-branch
+          value: '{{ target_branch }}'
+        - name: oci-storage
+          value: $(params.output-image).sealights.git
+    - name: sealights-unit-tests
```

Comment:
We currently run unit tests in Github workflows: https://github.com/konflux-ci/build-service/blob/273138dd021ce565cb5f1fd7ab674f729d0f93f4/.github/workflows/pr.yaml#L49-L50

Reply:
i would delete them and move to sealights. We will capture there the coverage

Reply:
sounds reasonable :+1: